src/benchmarks/gc/src/commonlib/document.py

```diff
@@ -139,6 +139,9 @@ def _pad(width: int, s: str, align: Align) -> str:
     return switch[align]()
 
 
+_SUPPORTED_TERMINALS = {"ConEmuC64.exe", "WindowsTerminal.exe"}
```

Comment:
did we actually figure why there was no color in any of the terminals? Do we need a bug logged on that?

Reply:
I'm curious about the same thing

---
In reply to: [355727790](https://github.com/dotnet/performance/pull/1086#discussion_r355727790) [](ancestors = 355727790)

Reply:
Actually, I should either remove this and leave the color fix on its own branch and PR, as it is right now, or I could merge it in this one. I believe the former is the better way to proceed, but I would like to hear your thoughts.

Reply:
Taking a closer look, this set was added to simplify the color function, but does not affect that functionality. That fix will be added in another PR.